Assignment3.Rmd

```diff
@@ -174,7 +174,7 @@ A3<- tidyr::gather(A2, "location","geoinfo", 1:2)
 A4<- A3 %>% group_by(location, cluster)
 A5<- summarise(A4, avg=mean(geoinfo))
 A5$cluster <- as.factor(A5$cluster)
-ggplot(A5,aes(location, avg, colour=cluster))+geom_line()
+ggplot(A5,aes(location, avg, colour=cluster))+geom_dotplot()
 ```
 #Part III Create a visualization that shows the overlap between two groups of clusters you created in partIII.
 ```{r}
```

Comment:
#visualize the data
library(ggplot2)

ggplot(aes(x='location', y='avg'), data=A2) + geom_point() + 
       geom_point(aes(x='question', y='avg'), data=D5)

Reply:
I updated the MacOS and git stopped working.
This is what I have for PART III.